src/app.es6.js

```diff
@@ -92,20 +92,24 @@ class App {
     this.emitter.removeListener.apply(this.emitter, args);
   }
 
-  error (err, ctx, app) {
-    var status = err.status || 500;
-    var message = err.message || 'Unkown error';
+  error (e, ctx) {
+    var status = e.status || 500;
+    var message = e.message || 'Unkown error';
     var url = '/' + status;
 
+    if (this.config.debug) {
+      console.log(e, e.stack);
+    }
+
     if (ctx.request.url !== url) {
       ctx.set('Cache-Control', 'no-cache');
       ctx.originalUrl = ctx.originalUrl || ctx.request.url;
       ctx.path = url;
```

Comment:
seems strange that we're setting things on the context and not using them directly